go.mod

```diff
@@ -5,6 +5,8 @@ go 1.13
 require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-openapi/spec v0.19.4
+	github.com/go-playground/assert/v2 v2.0.1
```

Comment:
https://github.com/stretchr/testify more common using for test assertions

Reply:
:+1: It's not actually used